sim/hunter/item_sets_pvp.go

```diff
@@ -128,3 +128,99 @@ var ItemSetLieutenantCommandersProwess = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+///////////////////////////////////////////////////////////////////////////
+//                            SoD Phase 5 Item Sets
+///////////////////////////////////////////////////////////////////////////
+
+var ItemSetWarlordsPursuit = core.NewItemSet(core.ItemSet{
```

Comment:
It looks like the gloves of the Pursuit sets have +4% damage to multi-show as well. That can be done in `sim/hunter/items.go` for both pairs
<img width="404" alt="image" src="https://github.com/user-attachments/assets/fe5d7bef-5150-4e77-963a-de9cd08ecea9">